iron-selectable.html

```diff
@@ -132,6 +132,8 @@
     attached: function() {
       this._observer = this._observeItems(this);
       this._contentObserver = this._observeContent(this);
+      this._removeListener(this.activateEvent);
```

Comment:
@Collaborne since this is already removed in `detached`, can you explain why it is removed again here?

Also, it would be nice to have a test to catch future regressions.